### PR TITLE
Switch assets UI

### PR DESCRIPTION
**Pull request by Daniel-Wang**
### Features

- Complete the heterogeneous recyclerview for AssetsActivity
- Use Picasso to load images
- Update documentation on model classes

### Screenshots

screenshot_20180928-144420

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -37,19 +37,26 @@ dependencies {
...
     implementation "com.android.support.constraint:constraint-layout:$constrainLayoutVersion"
     implementation 'com.android.support:multidex:1.0.3'
+    implementation 'com.android.volley:volley:1.1.1'
 
     implementation "android.arch.lifecycle:extensions:$lifecycle_version"
     kapt "android.arch.lifecycle:compiler:$lifecycle_version"
 
     implementation "com.google.android:flexbox:$flexboxVersion"
 
+    implementation ('com.squareup.picasso:picasso:2.71828') {
```
Comment (MeghaBambra) [inline app/build.gradle]: Curious - this is to exclude duplicate dependencies?

File: app/src/main/java/blockeq/com/stellarwallet/models/SupportedAsset.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package blockeq.com.stellarwallet.models
+
+/**
+ * Class which provides a model for SupportedAsset
```
Comment (MeghaBambra) [inline app/src/main/java/blockeq/com/stellarwallet/models/SupportedAsset.kt]: 💯 for documentation

**Comment by MeghaBambra** [comment]
@Daniel-Wang LGTM :shipit:

File: app/build.gradle (hunk 1 above)
Comment (Daniel-Wang) [reply app/build.gradle]: Yeah, picasso uses duplicate dependencies internally so i need to exclude them to build properly

**Pull request merged**